elpis/endpoints/model.py

```diff
@@ -86,32 +97,18 @@ def settings():
     })
 
 
-@bp.route("/l2s", methods=['POST'])
-def l2s():
-    m: Model = app.config['CURRENT_MODEL']
-    # handle incoming data
-    if request.method == 'POST':
-        file = request.files['file']
-        if m is None:
-            # TODO some of the end points (like this one) return files, but on error we still return a json string? Looks like bad practice to me
-            return '{"status":"error", "data": "No current model exists (prehaps create one first)"}'
-        m.set_l2s_fp(file)
-    return m.l2s
-
-
-@bp.route("/lexicon", methods=['GET', 'POST'])
-def generate_lexicon():
-    m: Model = app.config['CURRENT_MODEL']
-    m.generate_lexicon()
-    return m.lexicon
-
 
 @bp.route("/list", methods=['GET', 'POST'])
 def list_existing():
     kaldi: KaldiInterface = app.config['INTERFACE']
     # TODO see the two todos below
     fake_results = {}
-    lx = [{'name': model['name'], 'results': fake_results, 'dataset_name': model['dataset_name']} for model in kaldi.list_models_verbose()]
+    lx = [{
```

Comment:
This is a pre-existing problem, but this variable needs a better name.

Reply:
It could be `data`?